llettuce-docs/autoapi/app/index.rst

```diff
@@ -202,18 +240,31 @@ Module Contents
 .. py:function:: generate_events(request: PipelineRequest) -> collections.abc.AsyncGenerator[str]
    :async:
 
+   Generate LLM output and OMOP results for a list of informal medication names
 
-   Generate LLM output and OMOP results for an informal medication name
-
-   The first event is the reply from the LLM
-   The second event fetches relevant concepts from the OMOP database using the LLM output
+   Parameters
+   ----------
+   request: PipelineRequest
+       The request containing the list of informal names of the medications.
 
-   The function yields results as they become available, allowing for real-time streaming.
+   Workflow
+   --------
+   For each informal name:
+       The first event is to Query the OMOP database for a match
+       The second event is to fetches relevant concepts from the OMOP database
+       Finally,The function yields results as they become available,
```

Comment:
the function, instead of The function? As they become available from the database query or from the LLM matching?